gebuilder/scripts/initialize/default/00-copy.sh

```diff
@@ -1,5 +1,10 @@
 #!/bin/bash
 
+if [ ! -f  "/var/lib/gebuilder/roots/stemgentoo/root/" ]; then 
+       error "Stemgen was not initialized"
+       error exit
```

Comment:
`error exit` should be `error_exit`, since we do not want to call the error function again.